app.py

```diff
@@ -20,5 +24,16 @@ async def add_role(member: discord.Member, role_id: int):
     role: discord.Role = member.guild.get_role(config.ROLE_ON_JOIN_ID)
     await member.add_roles(role)
 
+@bot.event
+async def on_ready():
+    def message_cog_maker(prompt):
+        if(prompt['type']=='question'):
+            bot.add_cog(ScheduledMessage(bot, prompt['message'], prompt['channelID'], prompt['intervalSeconds']))
+    if (os.path.isfile('./messages.json')):
+        f = open('./messages.json', 'r')
```

Comment:
I would avoid using non-descriptive variable names like `f`. This could be something like `prompt_file`.